lesson08/assignment/video_cli/l8/package.json

```diff
@@ -37,7 +40,9 @@
     "parserOptions": {
       "parser": "@babel/eslint-parser"
     },
-    "rules": {}
+    "rules": {
+      "vue/multi-word-component-names": 0
```

Comment:
This is just a lint rule to prevent errors when you don't use multi-word filenames like top3.vue would have to be top3Page.vue or something like that.